### PR TITLE
(feat) (abc 243) list component actions display

### PR DESCRIPTION
**Pull request by AlainRusnov**
List actions : 
- 1 action no label : button-icon
- 1 action label : button
- multi actions : button menu

### CODE REVIEW

**Review by jbverger — CHANGES_REQUESTED** [review]

File: src/modules/base/list/list.html [hunk 1]
```diff
@@ -125,23 +125,52 @@ <h3>{item.label}</h3>
...
+                                <lightning-menu-item
+                                    key={action.name}
+                                    value={action.name}
+                                    label={action.label}
+                                    disabled={action.disabled}
+                                    icon-name={action.iconName}
+                                ></lightning-menu-item>
+                            </template>
+                        </lightning-button-menu>
+                        <template if:false={multiActions}>
```
Comment (nawel-aa) [inline src/modules/base/list/list.html:149]: You should not have two `template if` one inside each other, it's better to keep your logic inside your Javascript file (see [guidelines](https://avonnicreator.atlassian.net/wiki/spaces/LDG/pages/1587019781/HTML+Templates)).
Maybe you could use only`if:false={multiActions}`, and add a `if:true={actions.length}` to your `<div>` line 128, to handle the case of empty actions?

**Review by nawel-aa — CHANGES_REQUESTED** [review]
There's a small HTML syntax problem, but most importantly, there are no tests for the actions attribute.

File: src/modules/base/list/list.js [hunk 2]
```diff
@@ -130,6 +131,14 @@ export default class List extends LightningElement {
         this._hasActions = true;
     }
 
+    get singleAction() {
+        return this.computedActions[0];
+    }
+
+    get multiActions() {
```
Comment (jbverger) [inline src/modules/base/list/list.js:138]: Rename multiActions to hasMultipleActions

File: src/modules/base/list/list.js [hunk 3]
```diff
@@ -130,6 +131,14 @@ export default class List extends LightningElement {
         this._hasActions = true;
     }
 
+    get singleAction() {
```
Comment (jbverger) [inline src/modules/base/list/list.js:134]: Rename singleAction to firstAction.

**Review by jbverger — APPROVED** [review]

**Pull request merged**